testing/trino-benchto-benchmarks/src/test/java/io/trino/sql/planner/AbstractCostBasedPlanTest.java

```diff
@@ -54,6 +62,32 @@
 public abstract class AbstractCostBasedPlanTest
         extends BasePlanTest
 {
+    @Override
+    protected LocalQueryRunner createLocalQueryRunner()
+    {
+        String catalog = "local";
+        SessionBuilder sessionBuilder = getSessionBuilder()
+                .setCatalog(catalog)
+                .setSystemProperty("task_concurrency", "1") // these tests don't handle exchanges from local parallel
+                .setSystemProperty(JOIN_REORDERING_STRATEGY, JoinReorderingStrategy.AUTOMATIC.name())
+                .setSystemProperty(JOIN_DISTRIBUTION_TYPE, JoinDistributionType.AUTOMATIC.name());
+        LocalQueryRunner queryRunner = LocalQueryRunner.builder(sessionBuilder.build())
+                .withNodeCountForStats(8)
+                .build();
+        queryRunner.createCatalog(
+                catalog,
+                getConnectorFactory(),
+                ImmutableMap.of());
+        return queryRunner;
+    }
+
+    protected SessionBuilder getSessionBuilder()
```

Comment:
Rather than this it would be better to have a `protected  abstract String getSchema()`